documentation/shared/attributes.adoc

```diff
@@ -18,6 +18,7 @@
 :ContextProduct: strimzi
 :OpenShiftVersion: 3.11 and later
 :KubernetesVersion: 1.11 and later
+:Zookeeper: ZooKeeper
```

Comment:
Why do we need this variable? Writting the modular docs is quite hard even without yet another variable to follow. So I think it needs to have a good justification. But right now I do not see why would you need to use something else instead of `ZooKeeper`.

Reply:
Variables give us more flexibility and offer us a bit of protection. Should the name change, we'll be fine. If the consensus is that it's overkill, then I'll replace the attributes with the text.

Reply:
My view is:
* Your PR alone doesn't use it everywhere.
* Doing a search and replace in the whole docs for `ZooKeeper` to rename it to something else sounds quite easy.

I'm also really concerned about the friendliness of the documentation toward contributors. Obviously this is a small piece compared to all the other obstacles. But the more of them we put into the docs, the more work will rely on you and @laidan6000 and we will have the same issues as with 0.14 release where the docs are finished only week or two after the actual release.

Reply:
Understood. If the potential for inconsistency is less important than making the docs easy to update, we can use text.

Reply:
@laidan6000 - what are your thought? Stick with text? (Incidentally, I'm not sure we should be using Mirror Maker as two words either)

Reply:
I think we should stick with text for ZooKeeper as it's better to keep the source files as clean as possible. 

MirrorMaker seems to be the most used convention. I'm certainly guilty of using two separate words on a few occasions...

Reply:
@scholzj @laidan6000 - Thanks. We're back to text. :thumbsup: